packages/lesswrong/components/comments/CommentsItem/CommentsItemDate.tsx

```diff
@@ -37,13 +36,13 @@ const styles = (theme: ThemeType): JssStyles => ({
   link: {
   },
   icon: {
-    fontSize: "0.9rem",
-    transform: "rotate(-45deg)",
+    fontSize: "0.9rem !important",
+    transform: theme.uiStyle === "friendly" ? undefined : "rotate(-45deg)",
```

Comment:
This is interesting - since the rotation is specific to a single icon, maybe this style should be applied by `ForumIcon` in that specific case, instead of passed in by the caller? That way we won't have to depend on LW remembering that the rotation only makes sense for their own icon.